packages/lisk-passphrase/src/validation.ts

```diff
@@ -117,9 +117,9 @@ export const locateConsecutiveWhitespaces = (
 export const getPassphraseValidationErrors = (
 	passphrase: string,
 	wordlists?: ReadonlyArray<string>,
+	expectedWords: number = 12,
```

Comment:
should this be optional: `expectedWords?: number = 12`

Reply:
once we have default value, it doesn't allow to use optional. Instead it will automatically read it as optional